graphrag/index/__main__.py

```diff
@@ -4,73 +4,123 @@
 """The Indexing Engine package root."""
 
 import argparse
+from enum import Enum
+from pathlib import Path
 
 from .cli import index_cli
 
+
+class ReporterType(Enum):
+    """The type of reporter to use."""
+
+    RICH = "rich"
+    PRINT = "print"
+    NONE = "none"
+
+    def __str__(self):
+        """Return the string representation of the enum value."""
+        return self.value
+
+
+class EmitType(Enum):
+    """The type of emitter to use."""
+
+    PARQUET = "parquet"
+    CSV = "csv"
+
+    def __str__(self):
+        """Return the string representation of the enum value."""
+        return self.value
+
+
+def file_exist(path):
```

Comment:
THis looks like it could go into the top level utils module

Reply:
Done. Moved to the top-level utils directory.